src/neptune/internal/backends/api_model.py

```diff
@@ -132,6 +132,7 @@ class ClientConfig:
     _missing_features: FrozenSet[str]
```

Comment:
Let's leave a note in the Changelog. That's performance improvement so it's at least beneficial from an user perspective.

Reply:
done